packages/video_player/video_player_avfoundation/ios/Classes/Cache/Cache/CacheManager.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Foundation/Foundation.h>
+#import "CacheConfiguration.h"
+
+extern NSString *CacheConfigurationKey;
+extern NSString *CacheFinishedErrorKey;
+
+/*!
+ responsible for managing caching (directory) of content
+ */
+@interface CacheManager : NSObject
+
+/*!
+ returns filepath for file (content) url
+ */
++ (NSString *)cachedFilePathForURL:(NSURL *)url;
+
+/*!
+ returns CacheConfiguration for file (content) url
+ */
++ (CacheConfiguration *)cacheConfigurationForURL:(NSURL *)url error:(NSError **)error;
+
+/*!
+ removes all (downloading)files in cache directory
```

Comment:
Why would clearing the cache only clear in-progress downloads?

Reply:
it does not. I'll change the documentation, it seems unclear.